src/arrays/ArrayUtil.java

```diff
@@ -63,32 +115,62 @@ public static void remove(int[] array, int index) {
      * Calculate and return sum of array's elements.
      */
     public static long sum(int[] array) {
+        int sum = 0;
 
-        return 0;
+        for (int i = 0; i < array.length; i++) {
+            sum += array[i];
+        }
+
+        System.out.println(sum);
+        return sum;
     }
 
 
     /**
      * Get the max value from the array.
      */
     public static int getMax(int[] array) {
+        int maxValue = array[0];
+
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] > maxValue) {
+                maxValue = array[i];
+            }
+
+        }
+        System.out.println(maxValue);
 
-        return 0;
+
+        return maxValue;
     }
 
     /**
      * Get the minimum value from the array
      */
     public static void getMin(int[] array) {
+        int minValue = array[0];
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] < minValue) {
+                minValue = array[i];
+            }
+        }
+        System.out.println(minValue);
+
 
     }
 
     /**
      * Calculate average of array
      */
     public static int getAvg(int[] array) {
-
-        return 0;
+        int sum = 0;
+        int average = sum / array.length;
+        for (int i = 0; i < array.length; i++) {
+            sum += array[i];
+        }
+        System.out.println(sum / array.length);
+
+        return average;
     }
 
```

Comment:
The Average was 0 but you returned it.
You don't need to write sum() method because you already wrote it, you must used it. 
For example`
 public static int getAvg(int[] arr){
        return sum(arr)/arr.length;
    }